test/spoom/cli/bump_test.rb

```diff
@@ -315,6 +315,36 @@ def test_bump_dry_does_nothing_with_no_bumpable_file
         assert_equal("false", Sorbet::Sigils.file_strictness("#{@project.path}/file2.rb"))
       end
 
+      def test_bump_dry_does_not_revert_files_not_bumped
+        @project.write("file1.rb", <<~RB)
+          # typed: false
+          class A
+            def foo(a, b); end
+          end
+        RB
+        @project.write("file2.rbi", <<~RB)
+          # typed: true
+          class A
+            def foo(a, b, c); end
+          end
+        RB
+
+        out, err, status = @project.bundle_exec("spoom bump --no-color --from false --to true --dry")
+        assert_empty(err)
+        assert_equal(<<~OUT, out)
+          Checking files...
+
+          Can bump `1` file from `false` to `true`:
+           + file1.rb
+
+          Run `spoom bump --from false --to true` to bump them
+        OUT
```

Comment:
I honestly don't know how to do so.

One way would be to parse the error message for `file2.rbi` and see that the first help section points to `file1.rb`. But the help sections points to a bad method more often than the other way around.

Any idea?

Reply:
Isn't it safe to assume if an already `true` file gave an `redefined without matching argument count` error it would have to be the file inside help section? We can also make sure that the file inside help section was bumped. 

This solution does assume `typecheck` passes before running spoom.

Reply:
As per our conversation on Slack: let's merge as this and improve later 👍